components/service/glean/README.md

```diff
@@ -119,14 +119,16 @@ for the command line switches used to pass the extra keys. These are the current
 |key|type|description|
 |---|----|-----------|
 | logPings | boolean (--ez) | If set to `true`, glean dumps pings to logcat; defaults to `false` |
-| sendPing | string (--es) | Sends the ping with the given name immediately. |
+| sendPing | string (--es) | Sends the ping with the given name immediately |
+| debugID | string (--es) | Tags all pings using the `X-Debug-ID` header combined with the given value |
```

Comment:
Nit: Why not follow camel-case consistently here?

Reply:
After reading the debug view doc again, I renamed this to `tagPings` as seen in the doc (also observing camelCase :)